src/op_analytics/transforms/fees/update/04_agg_daily_trace_to.sql

```diff
@@ -0,0 +1,257 @@
+  , SUM(tx_estimated_size) AS sum_tx_estimated_size_called
+
+  -- Sum Internal Trace Gas
+  /*
+    Note: As opposed to transaction-level equivalent metrics, these CAN be summed up at a project level.
+    These methodologies were designed to distribute gas usage and fees across each call, so that they
+    still represent a "part of a whole," whereas transaction-level metrics do not.
+  */
+
+  , SUM(preagg_sum_trace_gas_used) AS sum_trace_gas_used
+  , sumIf(preagg_sum_trace_gas_used, tx_success) AS sum_trace_gas_used_tx_success
+
+  , SUM(preagg_sum_trace_gas_used_minus_subtraces) AS sum_trace_gas_used_minus_subtraces
+  , sumIf(preagg_sum_trace_gas_used_minus_subtraces, tx_success) AS sum_trace_gas_used_minus_subtraces_tx_success
+
+  , SUM(preagg_sum_tx_l2_gas_used_amortized_by_call) AS sum_tx_l2_gas_used_amortized_by_call
+  , sumIf(preagg_sum_tx_l2_gas_used_amortized_by_call, tx_success) AS sum_tx_l2_gas_used_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l1_gas_used_unified_amortized_by_call) AS sum_tx_l1_gas_used_unified_amortized_by_call
+  , SUM(preagg_sum_tx_l1_base_scaled_size_amortized_by_call) AS sum_tx_l1_base_scaled_size_amortized_by_call
+  , SUM(preagg_sum_tx_l1_blob_scaled_size_amortized_by_call) AS sum_tx_l1_blob_scaled_size_amortized_by_call
+  , SUM(preagg_sum_tx_estimated_size_amortized_by_call) AS sum_tx_estimated_size_amortized_by_call
+
+  , SUM(preagg_sum_tx_l2_fee_native_minus_subtraces) AS sum_tx_l2_fee_native_minus_subtraces
+  , sumIf(preagg_sum_tx_l2_fee_native_minus_subtraces, tx_success) AS sum_tx_l2_fee_native_minus_subtraces_tx_success
+
+  , SUM(preagg_sum_tx_l2_base_fee_native_minus_subtraces) AS sum_tx_l2_base_fee_native_minus_subtraces
+  , SUM(preagg_sum_tx_l2_priority_fee_native_minus_subtraces) AS sum_tx_l2_priority_fee_native_minus_subtraces
+  , SUM(preagg_sum_tx_l2_legacy_base_fee_native_minus_subtraces) AS sum_tx_l2_legacy_base_fee_native_minus_subtraces
+
+  , SUM(preagg_sum_tx_fee_native_amortized_by_call) AS sum_tx_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_fee_native_amortized_by_call, tx_success) AS sum_tx_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l2_fee_native_amortized_by_call) AS sum_tx_l2_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_l2_fee_native_amortized_by_call, tx_success) AS sum_tx_l2_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l1_fee_native_amortized_by_call) AS sum_tx_l1_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_l1_fee_native_amortized_by_call, tx_success) AS sum_tx_l1_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l2_base_fee_native_amortized_by_call) AS sum_tx_l2_base_fee_native_amortized_by_call
+  , SUM(preagg_sum_tx_l2_priority_fee_native_amortized_by_call) AS sum_tx_l2_priority_fee_native_amortized_by_call
+
+  , SUM(preagg_sum_tx_fee_native_l1_amortized_l2_minus_subtraces) AS sum_tx_fee_native_l1_amortized_l2_minus_subtraces
+
+  -- Calls
```

Comment:
We can cut the `trace_success` variants here: `count_internal_calls_all_types_trace_success` through `count_internal_calls_call_type_trace_success`